ensime_shared/config.py

```diff
@@ -55,7 +56,8 @@
     "split_window": "split {}",
     "vert_split_window": "vsplit {}",
     "new_vertical_window": "{}vnew {}",
-    "new_vertical_scratch": "{}vnew {} | setlocal nobuflisted buftype=nofile bufhidden=wipe noswapfile",
+    "new_vertical_scratch":
+        "{}vnew {} | setlocal nobuflisted buftype=nofile bufhidden=wipe noswapfile",
```

Comment:
I like the idea of an editor object that exposes most of the vim functionality we use.